src/routes/lobby/LobbyView.vue

```diff
@@ -153,15 +157,34 @@ async function ready() {
           gameId: err.gameId,
         },
       });
+    } else {
+      const key = err?.message;
+
+      const msg = t(key);
+      showSnackbar(msg, 'error');
     }
+
+
+    
   }
   readying.value = false;
 }
 
+function showSnackbar(message, color = 'surface-2') {
+  snackbarMessage.value = message;
+  snackbarColor.value = color;
+  snackbarOpen.value = true;
+}
+
 async function setIsRanked() {
   await gameStore.requestSetIsRanked({
     isRanked: gameStore.isRanked,
   });
+
+  showSnackbar(
+    `${t('lobby.rankedChangedAlert')} ${gameStore.isRanked ? t('global.ranked') : t('global.casual')}`,
+    'surface-2'
+  );
```

Comment:
If the socket event needs to handle the snackbar for the case where the opponent makes the change and that same event is fired when the current user makes the request here, we can handle it solely in the socket handler and it will show up regardless of which user makes the request